### ISSUE TITLE
Formatting and Linting Not Working in VSCode Starting with Version 2024.5.0

### ISSUE THREAD

**Issue by Dresdn** [djlint/djlint-vscode#741]
Starting with version 2024.5.0 (and now 2024.5.1), both formatting and linting functionalities are not working within VSCode. When I downgrade to version 2023.11.0, everything works as expected.

I'm not seeing any output or anything indicating an error, so please let me know what additional information is needed to help diagnose.

Thank you!

#### Additional Info

- The Output -> djlint panel is empty when attempting to format or lint.
- The Output -> Extension Host contains the following log:
```
[info] ExtensionService#_doActivateExtension monosans.djlint, startup: false, activationEvent: ‘onStartupFinished'
```
- No changes were made to `.vscode/settings.json` between versions.
- I tried setting `"djlint.useVenv": true` in the latest version, but it had no effect.

#### Versions

VScode: 1.86.0
djlint: 1.34.1
Python: 3.11.9

**Comment by monosans** [comment]
Hi! What is your OS?

**Comment by Dresdn** [comment]
D'oh! I'm on MacOS 14.4

**Comment by monosans** [comment]
These versions differ only in the [execa](https://github.com/sindresorhus/execa) and [esbuild](https://github.com/evanw/esbuild) updates.

As far as I can tell, this problem only occurs on macOS, and I don't have access to a macOS device. There are 2 options for me: write automated tests and run them in GitHub Actions or run macOS in Docker.

**Comment by Dresdn** [comment]
I noticed the issue as well and investigated `execa` and found that version 9.0.0 [dropped support for Node <18.19.0](https://github.com/sindresorhus/execa/releases/tag/v9.0.0).

According to the [table of VSCode versions](https://github.com/ewanharris/vscode-versions), the latest version of Node shipped with VSCode is 18.18.2. This should mean that `execa` isn't compatible...

However, I realized I hadn't updated VSCode in some time, so I decided to try that. Updating to v1.89.1 resolved the problem. Interestingly, despite what the version table suggests, `execa` works with this setup. ¯\\_(ツ)_/¯
